lib/matplotlib/axes/_base.py

```diff
@@ -350,7 +350,7 @@ def _plot_args(self, tup, kwargs):
         # to one element array of None which causes problems
         # downstream.
         if any(v is None for v in tup):
-            raise ValueError("x and y must not be None")
+            raise ValueError("x, y, and format string must not be None")
```

Comment:
Huh, you shouldn't have got here according to the logic above.... I think you've actually found a bug, not a documentation error...